models/models.go

```diff
@@ -10,17 +10,6 @@ import (
 )
 
 const (
-	StepRunStatusCodeSuccess           = 0
-	StepRunStatusCodeFailed            = 1
-	StepRunStatusCodeFailedSkippable   = 2
-	StepRunStatusCodeSkipped           = 3
-	StepRunStatusCodeSkippedWithRunIf  = 4
-	StepRunStatusCodePreparationFailed = 5
-	// StepRunStatusAbortedWithCustomTimeout is used when a step times out due to a custom timeout
-	StepRunStatusAbortedWithCustomTimeout = 7
-	// StepRunStatusAbortedWithNoOutputTimeout is used when a step times out due to no output received (hang)
-	StepRunStatusAbortedWithNoOutputTimeout = 8
-
 	// Version ...
 	Version = "12"
```

Comment:
Should we give this constant a better name?

Reply:
[4e60bd9](https://github.com/bitrise-io/bitrise/pull/836/commits/4e60bd92edc1e44241c4b121b63d5b949eed71d7)